controllers/news-controller.js

```diff
@@ -96,13 +86,20 @@ const { selectUserByUsername } = require('../models/users-model.js');
         res.status(204).send()
     })
     .catch(next);
-  }
+}
 
-  exports.getCommentById = (req, res, next) => {
+exports.getCommentById = (req, res, next) => {
     const {comment_id} = req.params
     selectCommentByCommentId(comment_id)
     .then((comment) => {
         res.status(200).send(comment)
     })
     .catch(next);
+}
+
+exports.getUsers = (req, res, next) => {
+    selectUsers()
+    .then((users) => {
+        res.status(200).send({users})
+    })
```

Comment:
I would consider splitting out this file because it's getting pretty long! You could have a controller for articles/topics/comments etc!